assets/js/mbta.ts

```diff
@@ -2762,27 +2811,12 @@ const stationConfig: {
         },
       },
       {
-        id: 'GLEC',
-        name: 'Lechmere',
-        zonePositions: {
-          left: ['n'],
-          center: [],
-          right: ['m'],
-        },
+        id: 'SLEC',
+        name: 'Lechmere - GLX',
```

Comment:
Should this just be `Lechmere`? It appears the other busway sign definitions use the plain station name, like `Nubian` or `Harvard`.

Reply:
Oh yeah, we should be able to do that since we deleted the old Lechmere busway!